### ISSUE TITLE
Error: Error building site: TOCSS: failed to transform "main.scss" (text/x-scss): resource "scss/styles/index.scss_a4e369375f17bb89f2019364cbe7a830" not found in file cache

### ISSUE THREAD

**Issue by huibaigu** [okkur/syna#729]
**Is this a BUG REPORT or FEATURE REQUEST?**:
bug
**What happened**:
it tell me "Error: Error building site: TOCSS: failed to transform "main.scss" (text/x-scss): resource "scss/styles/index.scss_a4e369375f17bb89f2019364cbe7a830" not found in file cache"
**What you expected to happen**:
i don't know
**How to reproduce it (as minimally and precisely as possible)**:
"hugo new site "
"hugo new aboud.md"
"cd themes"
"git clone https://github.com/okkur/syna.git"
"cd .."
"hugo server --theme=syna --buildDrafts"---when this it.
**Anything else we need to know?**:
none
**Environment**:
- Syna Theme version:  v0.16.2
- Hugo version:  v0.68.3
- Others:none

**Comment by stp-ip** [comment]
Are you using the Hugo extended version?

Just tried the below and this seems to work.
```
git clone https://github.com/okkur/syna.git
cd exampleSite
hugo server
```

**Comment by huibaigu** [comment]
i download hugo_0.68.3_Windows-64bit.zip in https://github.com/gohugoio/hugo/releases

**Comment by stp-ip** [comment]
In general hugo_extended_0.68.3_Windows-64bit.zip is necessary for Syna. That should resolve this issue.

Furthermore we have some issues with Windows support for fallback, which probably will come up. Linux is the preferred environment to use. Contributions to add windows support are definitely appreciated. Relate issue #630 
Last time we checked this was due to a windows specific issue in upstream Hugo, so not much we can do about this.
